plugin/gitsessions.vim

```diff
@@ -65,37 +65,45 @@ function! s:os_sep()
 endfunction
 
 function! s:is_abs_path(path)
-    return a:path[0] == s:os_sep()
+    if has('unix')
+        return a:path[0] == s:os_sep()
+    else
+        return a:path[1] == ':'
```

Comment:
Instead of hardcoding the path separator could we simply change the index position? Why use `== ':'`?

```suggestion
        return a:path[1] == s:os_sep()
```